bower.json

```diff
@@ -8,7 +8,7 @@
     "angular-ui-date": "~0.0.3",
     "angular": "~1.6.7",
     "bootstrap-css": "~3.0.0",
-    "jquery": "~2.1.0",
+    "jquery": "~3.2.1",
```

Comment:
You made sure that the updated version of jQuery being used doesnt conflict with anything else right?

Reply:
Correct, Tosin tested, and I also tested. jQuery is good for backwards compatibility.